jobs/periodic/drop-old-notification-contexts/index.ts

```diff
@@ -0,0 +1,24 @@
+import {prisma} from "../../../common/prisma";
+import moment from "moment";
+import {getLogger} from "log4js";
+
+const DELETE_AFTER = 30 * 3; // in days
+const logger = getLogger();
+
+export default async function execute() {
+
+    const res = await prisma.concrete_notification.updateMany({
+        where: {
```

Comment:
status: NotificationState.SENT, should be the case for all notifications but just want to be sure that no pending notification gets redacted accidentially before its sent out (which will then probably crash Mailjets template processing)